src/nmodl/main.cpp

```diff
@@ -251,6 +254,9 @@ int main(int argc, const char* argv[]) {
     codegen_opt->add_flag("--force",
         force_codegen,
         "Force code generation even if there is any incompatibility");
+    codegen_opt->add_flag("--opt-ionvar-cp",
```

Comment:
```suggestion
    codegen_opt->add_flag("--opt-ionvar-copy",
```

`nitpick` : copy might be better?